cypress/e2e/tests/migration/applicationinventory/analysis/downloaded_report.test.ts

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
File can be renamed to static_report.test.ts  to easily distinguish from other test download_report.test.ts